src/main/java/uk/gov/hmcts/reform/sandl/snlrules/model/now/Day.java

```diff
@@ -14,7 +14,7 @@
 @NoArgsConstructor
 @SuppressWarnings("squid:S1700")
```

Comment:
this needs to be removed now as the day field does not have the same name as the class, also all other